quickstarts/microsoft.compute/vm-simple-windows/main.bicep

```diff
@@ -100,15 +63,39 @@ param location string = resourceGroup().location
 @description('Name of the virtual machine.')
 param vmName string = 'simple-vm'
 
+@description('Security Type of the Virtual Machine.')
+@allowed([
+  'Standard'
+  'TrustedLaunch'
+])
+param securityType string = 'TrustedLaunch'
+
+@description('Secure Boot setting of the virtual machine. Applicable for securityType TrustedLaunch.')
+param secureBoot bool = true
+
+@description('Custom Attestation Endpoint to attest to. By default, MAA and ASC endpoints are empty and Azure values are populated based on the location of the VM.')
+param maaEndpoint string = ''
+
 var storageAccountName = 'bootdiags${uniqueString(resourceGroup().id)}'
 var nicName = 'myVMNic'
 var addressPrefix = '10.0.0.0/16'
 var subnetName = 'Subnet'
 var subnetPrefix = '10.0.0.0/24'
 var virtualNetworkName = 'MyVNET'
 var networkSecurityGroupName = 'default-NSG'
+var securityProfileJson = {
+  uefiSettings: {
+    secureBootEnabled: secureBoot
```

Comment:
What is the reasoning behind paramterizing secureboot and not vtpm? Why not keep it consistent, either have both as params or don't expose either as params. I would prefer not having either as params so that sb and vtpm are on by default.

Reply:
Turning On Secure boot has impact on bootchain as it will prohibit VM boot if boot chain is not attested. Providing parameter to users if they need to turn it OFF in their local copy of template.
vTPM is additional functionality and pre-requisite for modern OS like Windows 11. It does not impacts the boot process of VM.

Reply:
The user can just set the secureBootEnabled to false manually in their local copy. By parameterizing it will just add one more thing for the user to decide.

On a side note, secureboot will prohibit a VM from booting if there are unsigned kernel/modules in the boot chain but it does not rely on attestation.

Reply:
@sirfamelin - Replaced secureBoot parameter. Kindly review, also review the guest attestation extension condition.